omod/src/main/webapp/resources/htmlforms/hts/htsConfirmatoryTest.html

```diff
@@ -1837,11 +1864,29 @@
                 </tr>
             </table>
         </fieldset>
+        <fieldset id="hts-recency-testing">
+            <legend>Recency Testing</legend>
+            <table id="tbl-recency-testing">
+                <tr>
+                    <td>Has recency test been done? &#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;
+                        <obs id="recency-testing" conceptId="163777AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                             answerConceptIds="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                             answerLabel="Yes,No"
+                             style="radio" />
+                    </td>
+                </tr>
+                <tr id="tbl-recency-id">
+                    <td> Enter Recency Id :&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;
```

Comment:
Remove the word *Enter*. Just have it as Recency ID

Reply:
Okay  ...thanks